src/app/client/src/app/modules/dashboard/components/course-progress/course-progress.component.ts

```diff
@@ -153,6 +155,15 @@ export class CourseProgressComponent implements OnInit, OnDestroy, AfterViewInit
     * To get url, app configs
     */
   public config: ConfigService;
+  /**
+    * To display score report updated date
+    */
+  public scoreReportUpdatedOn: string;
```

Comment:
Why it is public?

Reply:
We are using this in html so this is public variable